src/i18n/en-US/businessCase.ts

```diff
@@ -0,0 +1,3 @@
+const businessCase = {};
```

Comment:
The empty files represent examples of namespaces where we can keep text. We don't need to stick to these namespaces. This should probably be decided outside of this story.